pubsub/test/iam.test.js

```diff
@@ -20,18 +20,19 @@ var policyMock = 'policy';
 var permissionsMock = 'permissions';
 
 function getSample () {
+  var apiResponseMock = {};
   var subscriptionMock = {
     iam: {
       getPolicy: sinon.stub().callsArgWith(0, null, policyMock),
```

Comment:
Add `apiResponseMock` to this for consistency - or do we care if it's not used in anything that changed?